pkg/rke2/rc.go

```diff
@@ -0,0 +1,86 @@
+package rke2
+
+import (
+	"context"
+	"sync"
+
+	"github.com/k3s-io/k3s/pkg/cli/cmds"
+	"github.com/k3s-io/k3s/pkg/util"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	runtimeClassesChart = "rke2-runtimeclasses"
+	defaultNamespace    = "kube-system"
```

Comment:
You don't need this, also it's not correct (as this is the system namespace, not default) - use the existing const `metav1.NamespaceSystem`